django_fido/locale/cs/LC_MESSAGES/django.po

```diff
@@ -50,6 +50,12 @@ msgstr "Neodstanitelné USB zařízení"
 msgid "Operation wasn't completed."
 msgstr "Operace nebyla dokončena."
 
+#, python-format
+msgid ""
+"Please enter a correct %(username)s and password and use valid FIDO2 security key."
+msgstr ""
+"Zadejte správnou hodnotu pole %(username)s a heslo a použijte validní FIDO2 klíč."
```

Comment:
Probably replace the first `a` with a comma? ` %(username)s, heslo a použijte ...`?

Reply:
Fixed.